Integration test failure on Circle

This seems to have come out of nowhere?

```
Traceback (most recent call last):

  File "/usr/local/lib/python3.6/dist-packages/py/_error.py", line 66, in checked_call

    return func(*args, **kwargs)

PermissionError: [Errno 13] Permission denied: '/code/app/tests/integration/.tox'



During handling of the above exception, another exception occurred:



Traceback (most recent call last):

  File "/usr/local/bin/tox", line 11, in <module>

    sys.exit(cmdline())

  File "/usr/local/lib/python3.6/dist-packages/tox/session/__init__.py", line 44, in cmdline

    main(args)

  File "/usr/local/lib/python3.6/dist-packages/tox/session/__init__.py", line 65, in main

    config.logdir.ensure(dir=1)

  File "/usr/local/lib/python3.6/dist-packages/py/_path/local.py", line 538, in ensure

    return p._ensuredirs()

  File "/usr/local/lib/python3.6/dist-packages/py/_path/local.py", line 520, in _ensuredirs

    parent._ensuredirs()

  File "/usr/local/lib/python3.6/dist-packages/py/_path/local.py", line 523, in _ensuredirs

    self.mkdir()

  File "/usr/local/lib/python3.6/dist-packages/py/_path/local.py", line 473, in mkdir

    py.error.checked_call(os.mkdir, fspath(p))

  File "/usr/local/lib/python3.6/dist-packages/py/_error.py", line 86, in checked_call

    raise cls("%s%r" % (func.__name__, args))

py.error.EACCES: [Permission denied]: mkdir('/code/app/tests/integration/.tox',)

Makefile:168: recipe for target 'integration_test' failed

make: *** [integration_test] Error 1


Exited with code exit status 2
```

--- a/.circleci/config.yml
+++ b/.circleci/config.yml
@@ -43,7 +43,7 @@ jobs:
       - run:
           name: Run integration tests
           command: |
-            sudo chown -R ubuntu app/tests/
+            sudo chmod -R a+rwx app/tests/
             make refresh
             make up_detached
             make integration_test

